adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -0,0 +1,131 @@
+package mobfoxpb
```

Comment:
Why did you choose `mobfoxpb` instead of simply `mobfox`?

Reply:
this comes from Prebid.js adapter, current adapters are made for new solution, but there is still working previous one which integrated through `mobfox` prebid.js adapter, so we decided to give new name to prebid.js and prebid-server adapters

Reply:
Gotcha. Sounds good for matching with a Prebid.js bidder name,